### PR TITLE
HDXDSYS-1085 Humanitarian needs table - replace gender, age, disability and population group columns with freeform category column

### PR DESCRIPTION
**Pull request by mcarans**
Javier requested we replace gender, age, disability and population group columns with the freeform category column because country offices do not have common standards so we can not reliably map that column to fixed gender, age, disability and population group values.

### CODE REVIEW

File: src/hapi_schema/db_humanitarian_needs.py [hunk 1]
```diff
@@ -18,15 +18,10 @@
 from hapi_schema.db_sector import DBSector
 from hapi_schema.utils.base import Base
 from hapi_schema.utils.constraints import (
-    max_age_constraint,
-    min_age_constraint,
     population_constraint,
     reference_period_constraint,
 )
 from hapi_schema.utils.enums import (
-    DisabledMarker,
```
Comment (turnerm) [inline src/hapi_schema/db_humanitarian_needs.py:27]: I think this can be deleted from the enums

**Review by turnerm — APPROVED** [review]
What a depressing change :C

File: src/hapi_schema/db_humanitarian_needs.py (hunk 1 above)
Comment (mcarans) [reply src/hapi_schema/db_humanitarian_needs.py:27]: I have done so. Yeah sure is unsatisfying making data less useful.

**Comment by IanHopkinson** [comment]
Is this OK to merge and publish as a version so that we can implement in HAPI/HWA? cc @turnerm

**Comment by turnerm** [comment]
I'll leave it to @mcarans as we still need to merge the pipeline branch

**Pull request merged**